frontend/common/src/helpers/router.tsx

```diff
@@ -1,6 +1,12 @@
 import { createBrowserHistory, Location, Path, State } from "history";
 import UniversalRouter, { Routes } from "universal-router";
-import React, { useState, useEffect, useMemo, ReactElement } from "react";
+import React, {
+  useState,
+  useEffect,
+  useMemo,
+  ReactElement,
+  HTMLAttributes,
+} from "react";
```

Comment:
seeing a linting thing here 😆

Reply:
I deleted that, what the heck lol 😭 

Resolved with [remove unused imports](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3027/commits/5bca08645f2ac36fa1a21175dd6ac790608bc6f4)